### PR TITLE
TASK: adding roles to account

### PR DESCRIPTION
**Pull request by mgoldbeck**

### CODE REVIEW

**Comment by mgoldbeck** [comment]
Not the best solution. Will try another

**Comment by Akii** [comment]
If you need support let me know!

File: Configuration/Settings.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ TYPO3:
           FacebookOAuth2Provider:
             provider: Flowpack\OAuth2\Client\Provider\FacebookProvider
             providerOptions:
+              #authenticateRoles: ['TYPO3.Flow.SomeRole']
               authorizationEndpointUri: 'https://www.facebook.com/dialog/oauth'
               redirectionEndpointUri: 'https://www.wishbase.com/authentication/authenticate'
```
Comment (afoeder) [inline Configuration/Settings.yaml:12]: haha; since you're on it @mgoldbeck could you nuke the `wishbase` occurrences in this repo please? :)

Comment (mgoldbeck) [inline Configuration/Settings.yaml:12]: will do ;)

**Pull request closed without merging**